delegation-verify-coordinator/templates/_helpers.tpl

```diff
@@ -0,0 +1,91 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "delegation-verify-coordinator.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "delegation-verify-coordinator.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Release.Name .Values.nameOverride }}
```

Comment:
Here should be
```
{{- $name := default .Chart.Name .Values.nameOverride }}
```
We discussed this in this pull request: https://github.com/MinaFoundation/helm-charts/pull/150/files/201a2975ba2529cb4b746763bfb3b369a60479b9#r1450041238